website/docs/en/blog/announcing-0-5.mdx

```diff
@@ -222,7 +222,7 @@ If you are using `@rspack/plugin-react-refresh`, now you need to install `react-
 
 ### Apply rspack.HotModuleReplacementPlugin
 
-If you are using `@rspack/cli`, or rsbuild, or other upper-level framework of Rspack to develop applications, you don't need to worry about this. This should be well handled by the upper-level framework or cli. But if you are using `@rspack/core` with a custom dev server (not `@rspack/dev-server` or `webpack-dev-server`), or developing a custom dev server, you need to notice this.
+If you are using `@rspack/cli`, or rsbuild, or other higher-level framework of Rspack to develop applications, you don't need to worry about this. This should be well handled by the higher-level framework or cli. But if you are using `@rspack/core` with a custom dev server (not `@rspack/dev-server` or `webpack-dev-server`), or developing a custom dev server, you need to notice this.
```

Comment:
`announcing-0-5.mdx` is not the focus of the current PR.

This PR has changes to that file, just because I globally replaced upper-level -> higher-level